src/analytics.js

```diff
@@ -119,7 +120,10 @@ export class Analytics {
 		document.querySelector('body').appendChild(script);
 
 		window.ga=window.ga||function(){(ga.q=ga.q||[]).push(arguments)};ga.l=+new Date;
-		ga('create', id, 'auto');
+		ga('create', id, 'auto', options);
+		for (var cmd of cmds) {
```

Comment:
Would it be better to have this follow the structure of the other GA features? `options` can stay as an arg for `init()`, but `cmds` will be moved to the `options` arg of `attach()`. For example:

`init(id, options) {...}`

```
attach(options = defaultOptions) {
    ...
    this._attachLinker(options.linker.cmds);
    this._attachClickTracker();
    this._attachPageTracker();
}
```

```
_attachLinker(cmds) {
    if(!this._options.linker.enabled) { return; }

    ga('require', 'linker');
    for (var cmd of cmds) {
        ga(cmd[0], cmd[1]);
    }
}
```